my-contacts/src/App.css

```diff
@@ -0,0 +1,10 @@
+body {
+  font-family: sans-serif;
+  overscroll-behavior-x: none;
+  overscroll-behavior-y: none;
+  height: 100%;
```

Comment:
We have a lot of things being 100% height to try and make things full screen

Reply:
Same for Font family also

Reply:
So we do, it can all go :)